### PR TITLE
add first entities + fixtures

### PR DESCRIPTION
**Pull request by mitch10593**

### CODE REVIEW

**Comment by mitch10593** [comment]
image

File: src/Entity/Cluster.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+class Cluster
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private ?int $id;
```
Comment (valAndre07) [inline src/Entity/Cluster.php:23]:
```suggestion
    private int $id;
```

File: src/Entity/Server.php [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+class Server
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private ?int $id;
```
Comment (valAndre07) [inline src/Entity/Server.php:23]:
```suggestion
    private int $id;
```

File: src/Entity/Server.php [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Cluster::class, inversedBy="servers")
+     * @ORM\JoinColumn(nullable=true)
+     */
+    private ?Cluster $cluster;
+
+    /**
+     * @ORM\Column(type="guid", nullable=true)
```
Comment (valAndre07) [inline src/Entity/Server.php:32]: Nullable ?

File: src/Entity/Slave.php [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+class Slave
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private ?int $id;
```
Comment (valAndre07) [inline src/Entity/Slave.php:21]:
```suggestion
    private int $id;
```

**Pull request merged**